Makefile

```diff
@@ -2,9 +2,11 @@ CARGO_HOME=$(HOME)/.cargo
 CARGO_BIN_DIR=$(CARGO_HOME)/bin
 CARGO_WATCH_DIR=$(CARGO_BIN_DIR)/cargo-watch
 CARGO_SYSTEMFD_DIR=$(CARGO_BIN_DIR)/systemfd
+CARGO_CLIPPY_DIR=$(CARGO_BIN_DIR)/cargo-clippy
```

Comment:
it's handy to add the cargo bin dir to ur PATH. E.g. my PATH contains: `/home/wilsoniya/.cargo/bin`